askama_shared/src/filters/mod.rs

```diff
@@ -75,6 +75,8 @@ pub const BUILT_IN_FILTERS: [&str; 27] = [
     "wordcount",
     "json", // Optional feature; reserve the name anyway
     "yaml", // Optional feature; reserve the name anyway
+    #[cfg(feature = "markdown")]
```

Comment:
If someone already has a custom markdown filter already, this PR would break their code.

Reply:
That's not really actually different from when the `json` and `yaml` filters were introduced, though... Also in the common case it probably wouldn't break their code since our implementation does the same thing anyway?